src/jorphan/src/main/java/org/apache/log/ContextMap.java

```diff
@@ -53,7 +54,7 @@
     private final ContextMap m_parent;
 
     ///Container to hold map of elements
-    private Hashtable m_map = new Hashtable();
+    private HashMap<String, Object> m_map = new HashMap<>();
```

Comment:
Are you sure synchronization is not required ?

Reply:
`ContextMap` was marked as `@deprecated Will be dropped in 3.3`.
Do we really need to investigate that?

I might drop the class altogether if you like.

Reply:
On the other hand, I should probably just suppress all the warnings for the class.